OpenSim/Simulation/Model/Muscle.h

```diff
@@ -623,9 +625,9 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Muscle, PathActuator);
         [1] fiberVelocityAlongTendon is the first derivative of the symbolic
             equation that defines the fiberLengthAlongTendon.
 
-        [2] normFiberVelocity is the fiberVelocity normalized with respect to 
-            the optimal fiber length of the fiber and the maximum fiber velocity
-            defined in fiber-lengths/s.
+        [2] normFiberVelocity is the fiberVelocity divided by  
+            the optimal fiber length of the fiber and by the maximum fiber velocity
+            expressed in optimal-fiber-lengths/s.
```

Comment:
A bit confusing. Perhaps:
"fiberVelocity" -> "fiberVelocity (in m/s)"
"optimal fiber length of the fiber" -> "optimal length of the fiber (in m)"
"maximum fiber velocity" -> "maximum fiber velocity (in optimal-fiber-lengths/s)"
And end with
"normFiberVelocity has units of 1/optimal-fiber-length."